### DISCUSSION TITLE
Can this be ran in Home Assistant?

### DISCUSSION THREAD

**Discussion by antonio1475** [Taxel/PlexTraktSync#827] [Q&A] [answered]
Hi. I just set it up on Windows and works great, thank you!

I have Plex Media Server in Home Assistant OS (maybe not ideal but works for my use case).
So, can PlexTraktSync be installed/ran in Home Assistant OS in any way? So that I don't depend on turning on the Windows computer to run the sync.

Apologies for the probably dumb question, I don't know Python at all. Thank you

**Comment by simonc56** [comment, marked as answer]
Maybe using the docker image [ghcr.io/taxel/plextraktsync](https://github.com/taxel/PlexTraktSync/pkgs/container/plextraktsync)

- 🎬 [Control Docker Containers with Home Assistant](https://www.youtube.com/watch?v=qYyLgu_7ADs)

**Reply by antonio1475** [reply]
Thank you. I ended up using Portainer as [described here](https://github.com/Taxel/PlexTraktSync/discussions/831).

**Comment by malfroid** [comment]
Instead of using a Portainer Add-on, would it be possible to create a new dedicated plextraktsync Add-on for Home Assistant OS following [this guide](https://developers.home-assistant.io/docs/add-ons/tutorial/)?

**Reply by glensc** [reply]
We accept pull requests

- https://github.com/Taxel/PlexTraktSync#contribute
